test/integration/README.adoc

```diff
@@ -0,0 +1,29 @@
+= CodeReady Containers Integration Tests
+:toc: macro
+:toc-title:
+:toclevels:
+
+toc::[]
+
+[[intro-to-integration-testing]]
+== Introduction
+Integration tests use http://github.com/code-ready/clicumber[Clicumber] package which provides basic functionality for testing CLI binaries.
+
+Clicumber allows to run commands in persistent shell instance (shell, tcsh, zsh, cmd, or powershell), assert its outputs (stdout, stderr or exitcode), check configuration files, etc.
```

Comment:
In this list of shell instances, is "shell" meant to refer to Bash?

```adoc
Clicumber allows running commands in a persistent shell instance (`bash`, `tcsh`, `zsh`, Command Prompt, or PowerShell), assert its outputs (standard output, standard error, or exit code), check configuration files, and so on.
```

Reply:
You're right, `bash` was meant... Thanks for suggesting alternatives with each comment! It should all be changed now.